File: TrabalhoC++/Dados.cpp
#include "Dados.hpp"

AlunoMap &Dados::getAlunos()
{
    return alunos;
}

void Dados::setAlunos(const AlunoMap &alunos)
{
    this->alunos = alunos;
}

CursoMap &Dados::getCursos()
{
    return cursos;
}

void Dados::setCursos(const CursoMap &cursos)
{
    this->cursos = cursos;
}

DisciplinaMap &Dados::getDisciplinas()
{
    return disciplinas;
}

void Dados::setDisciplinas(const DisciplinaMap &disciplinas)
{
    this->disciplinas = disciplinas;
}

AvaliacaoMap &Dados::getAvaliacoes()
{
    return avaliacoes;
}

void Dados::setAvaliacoes(const AvaliacaoMap &avaliacoes)
{
    this->avaliacoes = avaliacoes;
}

File: TrabalhoParteFinal/src/IO/Dados.java
package src.IO;

import java.io.Serializable;

import src.aluno.AlunoMap;
import src.avaliacao.AvaliacaoMap;
import src.curso.CursoMap;
import src.disciplina.DisciplinaMap;

public class Dados implements Serializable {
    AlunoMap alunos;
    CursoMap cursos;
    DisciplinaMap disciplinas;
    AvaliacaoMap avaliacoes;

    public AlunoMap getAlunos() {
        return alunos;
    }

    public void setAlunos(AlunoMap alunos) {
        this.alunos = alunos;
    }

    public CursoMap getCursos() {
        return cursos;
    }

    public void setCursos(CursoMap cursos) {
        this.cursos = cursos;
    }

    public DisciplinaMap getDisciplinas() {
        return disciplinas;
    }

    public void setDisciplinas(DisciplinaMap disciplinas) {
        this.disciplinas = disciplinas;
    }

    public AvaliacaoMap getAvaliacoes() {
        return avaliacoes;
    }

    public void setAvaliacoes(AvaliacaoMap avaliacoes) {
        this.avaliacoes = avaliacoes;
    }
}
